extensions/amazon-lambda-http/maven-archetype/src/main/resources/archetype-resources/src/test/java/GreetingTest.java

```diff
@@ -33,8 +38,44 @@ public void testVertx() {
     @Test
     public void testFunqy() {
         RestAssured.when().get("/funqyHello").then()
-                .contentType("text/plain")
-                .body(equalTo("hello funqy"));
+                .contentType("application/json")
+                .body(equalTo("\"hello funqy\""));
+    }
+
+    @Test
+    public void testGetTextJaxrs() throws Exception {
+        testGetText("/hello", "text/plain","hello jaxrs");
+    }
+
+    @Test
+    public void testGetTextServelet() throws Exception {
+        testGetText("/servlet/hello","text/plain", "hello servlet");
+    }
+
+    @Test
+    public void testGetTextVertx() throws Exception {
+        testGetText("/vertx/hello","text/plain", "hello vertx");
+    }
+
+    @Test
+    public void testGetTextFunqy() throws Exception {
+        testGetText("/funqyHello","application/json", "\"hello funqy\"");
+    }
+
```

Comment:
@patriot1burke 

FYI, there is an issue with the Funqy tests, it is using content type application/json and generates the below errors when run from the maven archetype (this needs to be fixed). I modified it here.

```
mvn archetype:generate \
        -DarchetypeGroupId=io.quarkus \
        -DarchetypeArtifactId=quarkus-amazon-lambda-http-archetype \
        -DarchetypeVersion=999-SNAPSHOT


[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.tjpower.GreetingTest
2020-06-04 10:35:43,346 INFO  [io.quarkus] (main) Quarkus 999-SNAPSHOT on JVM started in 1.833s. Listening on: http://0.0.0.0:8081
2020-06-04 10:35:43,347 INFO  [io.quarkus] (main) Profile test activated. 
2020-06-04 10:35:43,348 INFO  [io.quarkus] (main) Installed features: [amazon-lambda, cdi, funqy-http, mutiny, resteasy, servlet, vertx, vertx-web]
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.508 s <<< FAILURE! - in org.tjpower.GreetingTest
[ERROR] testFunqy  Time elapsed: 1.058 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected content-type "text/plain" doesn't match actual content-type "application/json".

	at org.tjpower.GreetingTest.testFunqy(GreetingTest.java:36)

2020-06-04 10:35:44,659 INFO  [io.quarkus] (main) Quarkus stopped in 0.102s
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   GreetingTest.testFunqy:36 1 expectation failed.
Expected content-type "text/plain" doesn't match actual content-type "application/json".

[INFO] 
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  17.379 s
[INFO] Finished at: 2020-06-04T10:35:45Z
[INFO] ————————————————————————————————————
```